lib/ensemble_provider.dart

```diff
@@ -60,8 +60,9 @@ class EnsembleDefinitionProvider extends DefinitionProvider {
     }
 
     if (content == null) {
-      throw LanguageError(
-          "Invalid screen content: ${screenId ?? screenName ?? 'Home'}");
+      throw LanguageError('invalid-content',
```

Comment:
We are saying this is NOT an error condition, so we should NOT be throwing error. Throwing an error can cause all sort of unknown and un-traceable issues.

You should refactor the code such that empty content is accepted by the system.

Reply:
@vusters I updated it, can you review it?